policies.md

```diff
@@ -0,0 +1,14 @@
+---
+layout: blue_page
+permalink: /info/
+title: Information & Policies
+---
+
+RustFest documents all their policies openly. They can be found here:
+
+* [Accessibility](/accessibility)
+* [Code of Conduct](/coc)
+    * We follow the PyCon [Attendee](https://us.pycon.org/2018/about/code-of-conduct/attendee-procedure/) and [Staff](https://us.pycon.org/2018/about/code-of-conduct/staff-procedure/) procedure for reporting
+* [Parents and nursing mothers](/parents)
+* [CFP submission and reimbursement policies](https://cfp.rustfest.eu)
```

Comment:
Do we have/will we have any more direct link for this?

Reply:
Not yet, the event is still private. I'll link to paris for now